src/renderer/tvgSaver.cpp

```diff
@@ -124,6 +139,32 @@ Result Saver::save(std::unique_ptr<Paint> paint, const string& path, bool compre
 }
 
 
+Result Saver::save(std::unique_ptr<Animation> animation, const string& path, int frameDelay) noexcept
```

Comment:
It appears to be duplicated code with the 'ResultSaver::save(std::unique_ptr<Paint> paint, const string& path, bool compress)' method. We can consolidate this method and reuse it.

Reply:
> We can consolidate this method and reuse it.
I have a question.
1) Is there a way to access the tvg::Animation object from the tvg::Picture(or Impl ?) object?
2) Do tvg::Animation and tvg::Paint need to inherit a new class? (ex tvg::Object ?)
3) compress parameter does not seem to be a valid option in tvg::Animation. Should I just leave it as an unsupported API?

Reply:
1. Not currently.
2. Probably, we can consider it when it proves genuinely effective for thorvg.
3. Yes, a compression parameter can be included to apply it optionally.

Never mind my previous feedback. I misunderstood the logic you implemented; the animation and picture couldn't be shared among the methods.